src/main/scala/com/typesafe/conductr/sbt/ConductR.scala

```diff
@@ -25,25 +26,36 @@ private[conductr] object ConductR {
   final val DefaultConductrProtocol = "http"
   final val DefaultConductrHost = "127.0.0.1"
   final val DefaultConductrPort = 9005
+
   val conductrAttrKey = AttributeKey[ActorRef]("sbt-conductr")
   val actorSystemAttrKey = AttributeKey[ActorSystem]("sbt-conductr-actor-system")
 
-  def loadBundle(bundle: URI, config: Option[URI], loadTimeout: Timeout, state: State): String = {
+  def loadBundle(pv: String, bundle: URI, config: Option[URI], loadTimeout: Timeout, state: State): String = {
     def get[A](key: SettingKey[A]) =
-      Project.extract(state).getOpt(key)
+      getOpt(key)
         .fold(Bad(One(s"Setting ${key.key.label} must be defined!")): A Or One[String])(Good(_))
-    def doLoadBundle(stm: String, roles: Set[String], nrOfCpus: Double, memory: Bytes, diskSpace: Bytes) = {
+    def getOpt[A](key: SettingKey[A]) =
+      Project.extract(state).getOpt(key)
+    def doLoadBundle(nn: String, v: String, stm: String, roles: Set[String], nrOfCpus: Double, memory: Bytes, diskSpace: Bytes) = {
```

Comment:
Well, `roles` or `nrOfCpus` are self explaining. But not `v` or `nn`. At least not for me :-)